extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -300,7 +300,11 @@ public void handle(Void e) {
             root = new Handler<HttpServerRequest>() {
                 @Override
                 public void handle(HttpServerRequest event) {
-                    delegate.handle(new ForwardedServerRequestWrapper(event, httpConfiguration.allowForwarded));
+                    delegate.handle(new ForwardedServerRequestWrapper(
+                            event,
+                            httpConfiguration.allowForwarded,
+                            httpConfiguration.forwardedHostHeader.getHeaderName(),
```

Comment:
How aboutr passing enums directly and only use `getHeader(Name)` when they needed, may be the enums will prove useful with some other checks...

Reply:
Already refactor it. However, I am not sure if I got the right idea. 
Could you check it again, please?

Reply:
Hi @ejba , sorry I missed this one, it looks fine. May be only have them as imports so as to avoid qualifying them with `HttpConfiguration.` inside the filter. 
This is probably quite subjective, my proposal to pass the parameters as enums :-), since other headers are `AsciiString`, but lets keep it as enum for now. I'll still ask Stuart to double check since he has done this code originally, so if you will be asked to revert back to passing `AsciiString` then I'm sorry :-)

Reply:
Hi @sberyozkin, that's fine! I will revert it if necessary. :-)